django_telegrambot/apps.py

```diff
@@ -170,11 +169,12 @@ def ready(self):
                 logger.error('WEBHOOK_CERTIFICATE not found in {} '.format(cert))
 
         for b in bots_list:
-            token = b.get('TOKEN', None)
-            context = b.get('CONTEXT', False)
-            if not token:
-                break
+            bot_data = {
+                'token': b['TOKEN'],
+                'id': b.get('ID', None)
```

Comment:
did you thought about fall back to the token when no id is given? If all other functions work with id=None thats fine, but if not it would be nice, to use some individual fallback, to not make breaking chanes to existing settings..

Reply:
Yes, I had in mind that my code shouldn’t break old code. The fallback for the ID are username and token, just like before.

Reply:
Actually the fallback for TOKEN are ID and username in order. because the methods have a safe attribute that shouldn't use anything other than the token, so It looks up the token first.